src/events/Windows.cpp

```diff
@@ -580,6 +580,11 @@ void Events::listener_unmapWindow(void* owner, void* data) {
 
     Debug::log(LOG, "{:c} unmapped", PWINDOW);
 
+    static auto PEXITRETAINSFS = CConfigValue<Hyprlang::INT>("misc:exit_window_retains_fullscreen");
+
+    const auto  pwindowCurrentFullscreenState = PWINDOW->m_bIsFullscreen;
+    const auto  pwindowCurrentFullscreenMode  = PWINDOW->m_pWorkspace->m_efFullscreenMode;
```

Comment:
style:
 these arent pointers, no `p` prefix
 consts are `CAPS`
 
`CURRENTWINDOWFSMODE` for example

Reply:
Ah, now I recall what the `p` is for. Haven't programmed in low-level languages in a few years oops.

I've fixed this up - thanks @vaxerski.

Happy to also make a PR for wiki if everything's good?